configuration/items.md

```diff
@@ -242,40 +258,80 @@ FIRST_QUARTER=🌓 First Quarter
 ```
 
 ## Icons
-OpenHAB provides you a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
+OpenHAB provides a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
 However if you wish to use custom icons you need to place them inside the `conf/icons/classic/` folder.
 These icons will be used in all of the openHAB frontends.
-The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required).
-The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
 
-As an example, to use a custom icon called `heatpump.svg` the correct syntax is `<heatpump>`.
+The images must be in `.png` or `.svg` format, consist of only lowercase letters, hyphens `-` or underscores `_` (if required).
```

Comment:
Testing... I would never do that but Nope, it's possible.

I've just successfully overwritten the "switch" icon by my own set of `icons/classic/switch.svg` + `icons/classic/switch{-off,-on}.svg`. After deleting `icons/classic/switch.svg` the state specific icons are now ignored, as reported above.

Maybe it is best to discourage the use of predefined iconset names in the custom icons to begin with!? Would probably be the cleanest advice here. If "extension" should be possible, we need to file a bug for that.